contrib/platform/test/com/sun/jna/platform/win32/MprTest.java

```diff
@@ -39,6 +39,12 @@ public static void main(String[] args) throws Exception {
         junit.textui.TestRunner.run(MprTest.class);
     }
 
+    public void testCreateLocalShare() throws Exception {
```

Comment:
_createLocalShare_ was just a private helper function for the other tests, so I wouldn't say it needs its own test. But of course, it can't hurt.